applications/GeoMechanicsApplication/tests/geo_beam_with_reset_displacement.gid/README.md

```diff
@@ -0,0 +1,23 @@
+# Staged analysis on a beam with reset displacement option
+
+This test asserts the 'reset_displacement' option of the GeoCrBeamElementLinear2D2N element works as expected. It exists of 4 stages, with different settings for 'reset_displacement' and applied load.
+
+## Setup
+The test describes a beam with a length of 1m, consisting of 11 equidistant nodes (0.0, 0.1, ... , 0.9, 1.0). The displacement and rotation of node 1 is fixed for all directions. In stage 1-3, a load in the negative y direction is applied to the right node (10), which is removed in stage 4. See the figure below for a schematic representation of the problem. 
+
+![MeshStructure](MeshStructure.svg)
+
+The following stages can be distinguished:
+1. A point load of 1e10 is applied to node 11 in the negative y-direction. The 'reset_displacement' option is set to true. The beam is expected to bend downwards.
+2. Essentially the same as stage 1: however, due to the 'reset_displacement' option, the displacement is expected to be 0.0 after this stage.
+3. The 'reset_displacement' option is set to false. The load is still the same.
+4. The load is removed and 'reset_displacement' is kept on false. The beam is expected to bend upwards again to the original position.
+
+## Assertions
+In stage 1, the displacement can be approximated using the following expression:
+$$u_y = \frac{FL^3}{3EI}L$$
+
+where $F$ is the applied force, $L$ is the length of the beam, $E$ is the Young Modulus and $I$ is the second moment of area. The displacement at the right node (10) is then $u_y \approx -0.016$ for this specific case.
+
+Due to the reset_displacement option, the displacement in stage 2 is expected to be 0.0 for all nodes. The same holds for stage 3 (is this expected, since 'reset_displacement' is set to false?). In stage 4, the load is removed, meaning that the displacement is expected to be the opposite of the displacement in stage 1 (bringing the beam back to its original position).
```

Comment:
The lack of movement in stages 2 and 3 is because the internal force is kept from stage 1 ( by the reset_displacement command in stages 1 and 2 ). This internal force is in equilibrium with the applied unchanging external force. Hence no movement.

Reply:
Thanks for the elaboration, changed!